modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java

```diff
@@ -191,6 +198,41 @@ public List<AccountGroupRel> getAccountGroupRelsByAccountGroupId(
 			accountGroupId, start, end, orderByComparator);
 	}
 
+	@Override
+	public List<AccountGroupRel> getAccountGroupRelsByAccountGroupName(
+		String keywords, long cpDefinitionId, int start, int end) {
```

Comment:
You need to implement this at the `service` layer not just, `localService`. Also, are there any examples of code that uses similar naming scheme as this?